cmd/juju/application/store/store.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2012, 2013 Canonical Ltd.
```

Comment:
Love this in in a store package, we should be able to do something for charmhub, so this is perfect naming.